apps/admin/frontend/src/screens/print_test_deck_screen.tsx

```diff
@@ -525,6 +523,18 @@ export function PrintTestDeckScreen(): JSX.Element {
         return handMarkedPaperBallotsWithCallback;
       });
 
+      const allTallyReportResults: Record<
```

Comment:
This method was previously returning, an asynchronous React component? I'm not sure, but linting didn't like it.